packages/ui-components/src/components/QrCode/QrCodeVisual.tsx

```diff
@@ -92,58 +100,52 @@ export const QrCodeVisual = ({
   if (!isEnlarged) {
     if (options.length > 1) {
       return (
-        <Container className={className} maxWidth="sm">
-          <p style={{ color: 'white', textAlign: 'center' }}>QR Code</p>
+        <StyledContainer className={className} maxWidth="sm">
+          <StyledParagraph>QR Codes</StyledParagraph>
```

Comment:
Since this is acting as a title, we should use the correct semantic HTML. Likely an h3, but depends on where it sits on the screen. E.g if there is no h2 anywhere above it, it should be h2, otherwise h3, and so on.
Also, can we name these more descriptively? E.g. instead of `StyledParagraph` it would be `Title` or something?